packages/twenty-front/src/modules/views/components/ViewBarFilterDropdownVectorSearchInput.tsx

```diff
@@ -0,0 +1,43 @@
+import { DropdownContent } from '@/ui/layout/dropdown/components/DropdownContent';
+import { DropdownMenuSearchInput } from '@/ui/layout/dropdown/components/DropdownMenuSearchInput';
+import { GenericDropdownContentWidth } from '@/ui/layout/dropdown/constants/GenericDropdownContentWidth';
+import { useVectorSearchFilterOperations } from '@/views/hooks/useVectorSearchFilterOperations';
+import { useVectorSearchInputState } from '@/views/hooks/useVectorSearchInputState';
+import { useLingui } from '@lingui/react/macro';
+import { useDebouncedCallback } from 'use-debounce';
+
+export const ViewBarFilterDropdownVectorSearchInput = ({
+  filterDropdownId,
+}: {
+  filterDropdownId: string;
+}) => {
+  const { t } = useLingui();
+  const { vectorSearchInputValue, setVectorSearchInputValue } =
+    useVectorSearchInputState(filterDropdownId);
+  const { applyVectorSearchFilter } = useVectorSearchFilterOperations();
+
+  const debouncedApplyVectorSearchFilter = useDebouncedCallback(
+    (value: string) => {
+      applyVectorSearchFilter(value);
+    },
+    500,
+  );
+
+  const handleSearchChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+    const inputValue = e.target.value;
+    setVectorSearchInputValue(inputValue);
+    debouncedApplyVectorSearchFilter(inputValue);
+  };
+
+  return (
+    <DropdownContent widthInPixels={GenericDropdownContentWidth.Medium}>
```

Comment:
@charlesBochet Nice to see that this new API seems to be used quite intuitively.